hack/windows/Prepare-Node.ps1

```diff
@@ -82,21 +90,24 @@ mkdir -force C:\var\log\kubelet
 mkdir -force C:\var\lib\kubelet\etc\kubernetes
 mkdir -force C:\etc\kubernetes\pki
 New-Item -path C:\var\lib\kubelet\etc\kubernetes\pki -type SymbolicLink -value C:\etc\kubernetes\pki\
-
+{{CONTAINER_RUNTIME}}
 $StartKubeletFileContent = '$FileContent = Get-Content -Path "/var/lib/kubelet/kubeadm-flags.env"
-$global:KubeletArgs = $FileContent.Trim("KUBELET_KUBEADM_ARGS=`"")
+$global:KubeletArgs = $FileContent.TrimStart(''KUBELET_KUBEADM_ARGS='').Trim(''"'')
+
+$global:containerRuntime = 
 
-$netId = docker network ls -f name=host --format "{{ .ID }}"
+if ($global:containerRuntime -eq "Docker") {
+    $netId = docker network ls -f name=host --format "{{ .ID }}"
 
-if ($netId.Length -lt 1) {
+    if ($netId.Length -lt 1) {
```

Comment:
Not introduced by this commit, but why we execute the same command twice in the script?

Reply:
We don't need host network for containerd and we can keep the creation of the host network inside Start Kubelet Script. Made the relevant change.